std/bigint.d

```diff
@@ -912,7 +912,7 @@ public:
 
     /**
         $(D toString) is rarely directly invoked; the usual way of using it is via
-        $(LINK2 std_format.html#format, std.format.format):
+        $(MREF std, format.format):
```

Comment:
This is not a module reference. Notice the `#...` part. Use `REF` instead: `$(REF format, std,format)`. Or just leave as is, since the focus of this PR is on `MREF`.